media-gfx/exif/files/exif-0.6.22-empty-string-check.patch

```diff
@@ -0,0 +1,38 @@
+Source: https://github.com/libexif/exif/commit/f6334d9d32437ef13dc902f0a88a2be0063d9d1c.patch
```

Comment:
I'm going to throw in a link to the Gentoo bug too. Thanks!

(I'll handle this on merge)